src/foam/u2/TableView.js

```diff
@@ -46,6 +49,14 @@ foam.CLASS({
   methods: [
     function addObj(obj) {
       var e = this.start('tr')
+          .on('click', function() {
+            this.tableView.selection = obj;
```

Comment:
In principle, yes. In practice, scrolling is still instant, so I'm not sure it's worth the complexity of determining the target row (and corresponding object) from the click event.